mesonbuild/modules/gnome.py

```diff
@@ -311,36 +311,6 @@ def _print_gdbus_warning() -> None:
                      mlog.bold('https://github.com/mesonbuild/meson/issues/1387'),
                      once=True)
 
-    def _get_dep(self, state: 'ModuleState', depname: str, native: bool = False,
-                 required: bool = True) -> Dependency:
-        kwargs = {'native': native, 'required': required}
-        # FIXME: Even if we fix the function, mypy still can't figure out what's
-        # going on here. And we really dont want to call interpreter
-        # implementations of meson functions anyway.
-        return self.interpreter.func_dependency(state.current_node, [depname], kwargs)  # type: ignore
-
-    def _get_native_binary(self, state: 'ModuleState', name: str, depname: str,
-                           varname: str, required: bool = True) -> T.Union[ExternalProgram, OverrideProgram, 'build.Executable']:
```

Comment:
In the modulestate function, we drop most of these from the return value type, and declare that it only returns ExternalProgram. This seems wrong; the function basically exists to possibly return one of the other two.

Reply:
Indeed, copied state.find_program() but it was wrong too. Fixed both now.